src/Groups/GroupDetails/GroupDetailsContainer.js

```diff
@@ -65,28 +86,23 @@ const GroupDetailsContainer = ({
         });
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [history, group, mutator.groupDetails],
-  );
-
-  const isLoading = !(
-    get(resources, ['groupDetails', 'hasLoaded']) &&
-    get(resources, ['currentFiscalYears', 'hasLoaded'])
+    [history, groupData.groupDetails, mutator.groupDetails],
```

Comment:
maybe it makes sense to remove `mutator.groupDetails` from deps? I'm not sure that we can guarantee it doesn't re-creates every render

Reply:
seems yes